receiver/kubeletstatsreceiver/testdata/pods.json

```diff
@@ -11,11 +11,11 @@
             "name":"kube-scheduler",
             "resources": {
               "requests": {
-                "cpu": "50m",
+                "cpu": "2m",
                 "memory": "10M"
               },
               "limits": {
-                "cpu": "100m",
+                "cpu": "4m",
```

Comment:
Why is this changed?

Reply:
I updated the test data to give us more interesting percentages to work with.  The previous values produced really small numbers when used with the values set for the first pod in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/testdata/stats-summary.json

Reply:
> I updated the test data to give us more interesting percentages to work with. 

I'm just having troubles finding where this change is reflected. Is there a test output?

Reply:
In the `scraper_test.go` file, these changes let me test that there was a value above 1 for `k8s.pod.cpu_request_utilization` and `k8s.container.cpu_request_utilization` but less than 1 for `k8s.pod.cpu_limit_utilization` and `k8s.container.cpu_limit_utilization`.

Reply:
I see. Thanks for clarifying. Given that `fakeRestClient` provides static values, I think we should generate pdata output and use it for validation the way we do it in most oh other tests. Maybe worth filing an issue with `good first issue` label for this.

Reply:
Ya that is a good idea. I'll open an issue

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27468